models/staging/stg_workday.yml

```diff
@@ -1340,7 +1340,7 @@ models:
       - name: is_benefits_effect
         description: '{{ doc("benefits_effect") }}'
 
-      - name: is_caesarean_section_birth
+      - name: caesarean_section_birth
```

Comment:
This is `is_caesarean_section_birth` in the staging model, should this be changed?

Reply:
Yeah it should. I still have some pending changes locally where this is updated. You'll see this corrected in my next push. I'll mark this PR out of draft once this is ready.